.github/workflows/ci.yml

```diff
@@ -93,20 +81,18 @@ jobs:
           tests/solidity/ethereum/*.sol, # PASSING
           "tests/solidity/ethereum/{abiEncoderV[12],abiencodedecode}", # PASSING
           "tests/solidity/ethereum/{accessor,arithmetics,asmForLoop,builtinFunctions,c99_scoping_activation.sol,cleanup,constantEvaluator}", # PASSING
-          "tests/solidity/ethereum/array/{[a-hj-z]*.sol,inline_*.sol,array_memory_allocation,concat,push,slices}", # PASSING
+          "tests/solidity/ethereum/array/{[a-z]*.sol,inline_*.sol,array_memory_allocation,concat,push,slices}", # PASSING
```

Comment:
This could be just:
```suggestion
          "tests/solidity/ethereum/array/{*.sol,inline_*.sol,array_memory_allocation,concat,push,slices}", # PASSING
```